### PR TITLE
build: initial stack setup

### PR DESCRIPTION
**Pull request by alehander92**
Adapted a bit: added our names, prints "kv", adapted README.md, followed https://docs.haskellstack.org/en/stable/#quick-start-guide

resolving #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.stack-work/
+*~
```
Comment (ilievs) [inline .gitignore]: It's a good idea to add newlines at the end of every file.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Copyright 2023 Slavey Iliev, Alexander Ivanov
```
Comment (ilievs) [inline LICENSE]: Prefer not to have my name on here

File: kv.cabal [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+-- This file has been generated from package.yaml by hpack version 0.36.0.
+--
+-- see: https://github.com/sol/hpack
+
+name:           kv
+version:        0.1.0.0
+description:    Please see the README on GitHub at <https://github.com/alehander92/kv#readme>
+homepage:       https://github.com/githubuser/kv#readme
+bug-reports:    https://github.com/githubuser/kv/issues
+author:         Slavey Iliev, Alexander Ivanov
```
Comment (ilievs) [inline kv.cabal]: Name not required.

File: kv.cabal [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+maintainer:     alehander42@gmail.com
+copyright:      2023 Slavey Iliev, Alexander Ivanov
```
Comment (ilievs) [inline kv.cabal]: Name not required.

File: package.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
+name:                kv
+version:             0.1.0.0
+github:              "githubuser/kv"
+license:             BSD-3-Clause
+author:              "Slavey Iliev, Alexander Ivanov"
```
Comment (ilievs) [inline package.yaml]: Name not required.

File: package.yaml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+maintainer:          "alehander42@gmail.com"
+copyright:           "2023 Slavey Iliev, Alexander Ivanov"
```
Comment (ilievs) [inline package.yaml]: Name not required.

File: kv.cabal [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+license:        BSD-3-Clause
+license-file:   LICENSE
+build-type:     Simple
+extra-source-files:
+    README.md
+    CHANGELOG.md
+
+source-repository head
+  type: git
+  location: https://github.com/githubuser/kv
```
Comment (ilievs) [inline kv.cabal:24]: Fix: wrong URL

Comment (alehander92) [reply kv.cabal:24]: i tried to fix it, but this is autogenerated: can't find githubuser in `stack.yaml`, not sure what to do with it: i'll leave it like this for now

**Review by ilievs — APPROVED** [review]
Looks good

**Pull request merged**